src/app/shared/data-storage.service.ts

```diff
@@ -109,7 +122,7 @@ export class DataStorage{
         this.userData.lastName = undefined;
         this.userData.password = undefined;
         this.userData.team = undefined;
-        this.userData.skills = undefined;
+        this.userData.genericSkills = undefined;
         this.loggedInUser.next(this.userData);
```

Comment:
Why do we need to set everything as undefined and pass it in the parameter .?
by default it should be undefined